exporter/f5cloudexporter/README.md

```diff
@@ -0,0 +1,42 @@
+# F5 Cloud Exporter
+
+Exports data via HTTP to [F5 Cloud](https://portal.cloudservices.f5.com/).
+
+Supported pipeline types: metrics, traces, logs
+
+> :construction: This exporter is in beta and configuration fields are subject to change.
+
+## Getting Started
+
+The following settings are required:
+
+- `endpoint` (no default): The URL where to send data to. See your F5 Cloud account for details.
+- `source` (no default): A unique identifier that is used to distinguish where this data is coming from. This is in 
```

Comment:
Is it obvious for F5 users how to set this? Otherwise, it's not so clear from this sentence.

Reply:
Great catch, @anuraaga! I'll update this to provide an example.